editor/src/components/editor/store/dispatch-strategies.spec.tsx

```diff
@@ -83,3 +123,589 @@ describe('interactionCancel', () => {
     expect(actualResult.newStrategyState.currentStrategy).toBeNull()
   })
 })
+
+const testStrategy: CanvasStrategy = {
+  name: 'Test Strategy',
+  isApplicable: function (
+    canvasState: InteractionCanvasState,
+    interactionSession: InteractionSession | null,
+    metadata: ElementInstanceMetadataMap,
+  ): boolean {
+    return true
+  },
+  controlsToRender: [],
+  fitness: function (
+    canvasState: InteractionCanvasState,
+    interactionSession: InteractionSession,
+    strategyState: StrategyState,
+  ): number {
+    return 10
+  },
+  apply: function (
+    canvasState: InteractionCanvasState,
+    interactionSession: InteractionSession,
+    strategyState: StrategyState,
+  ): StrategyApplicationResult {
+    return [wildcardPatch('permanent', { canvas: { scale: { $set: 100 } } })]
+  },
+}
+
+describe('interactionStart', () => {
+  it('creates the initial state with a simple test strategy', () => {
+    const editorStore = createEditorStore(
+      createInteractionViaMouse(
+        canvasPoint({ x: 100, y: 200 }),
+        { alt: false, shift: false, ctrl: false, cmd: false },
+        { type: 'BOUNDING_AREA', target: EP.elementPath([['aaa']]) },
+      ),
+    )
+    const actualResult = interactionStart(
+      [testStrategy],
+      editorStore,
+      dispatchResultFromEditorStore(editorStore),
+    )
+    expect(actualResult.newStrategyState).toMatchInlineSnapshot(`
+      Object {
```

Comment:
I fear that if we write object snapshots like this for `newStrategyState` and `interactionData`, it will be an absolute pain to change / refactor these once we have dozens of tests. 
could this test get away by only checking to see if the strategy was picked and the initial state was created? somthing more similar to how you are checking the patched and unpatched canvas scale

Reply:
I'd surmise these functions aren't likely to change a lot, if we check less of the data then we're making the tests less likely to catch actual breakages. Plus this is being tested with a test strategy that is local to the tests themselves so other strategies changing wont affect these tests at all.

Reply:
yeah I get it that this is a test strategy, I'm just concerned whether adding a new key to StrategyState or interactionData will be breaking dozens of tests in the new future